tasks/retrieval/msrvtt/msrvtt_test.py

```diff
@@ -0,0 +1,20 @@
+
+register_dataset = {"retrieval_msrvtt_dataset.py": "RetrievalMSRVTTDataset"}
+register_evaluator = {"video_retrieval_evaluator.py": "RetrievalEvaluator"}
```

Comment:
`KeyError: 'RetrievalEvaluator is already registered in evaluator at RetrievalEvaluator'`
Rename it to `RetrievalMSRVTTDataset`